### PR TITLE
Add/json to csv data 37

### PR DESCRIPTION
**Pull request by anotheredward**
Add tool to download existing data for organisations wanting to use the schema tool to import their data.

### CODE REVIEW

**Comment by ebuckley** [comment]
Just pushed a simple readme up, feel free to modify as you see fit Ed.

I'm happy with the code, focused more on testing than code review in this case, but happy to dive a bit deeper if there is value.

:shipit:

**Comment by camfindlay** [comment]
@anotheredward @ebuckley Found that the CLI script only returns the first 10 rows, we need to add a rows argument etc "&rows=200" where 200 could be any number passed in.

**Comment by anotheredward** [comment]
@camfindlay Would you like 200 rows as a default?

**Comment by camfindlay** [comment]
Sounds as good as any number as long as you can set it in an arguement, this was only used as it's more than the 142 results I expected from MoH.

**Comment by ebuckley** [comment]
I re-tested this locally clearing the same dataset.

It downloads 10/14 datasets.

I did capture a chunk of the logs though, will pass them over on slack.

**Comment by camfindlay** [comment]
Which 4 datasets didn't harvest over? Could be an encoding error or something like that?

**Comment by ebuckley** [comment]
@camfindlay Correction on my 10/14 number. it is actually 10/13. In my first comment, I include the links to the datasets that appear not to harvest correctly.
```
https://catalogue.data.govt.nz/dataset/lidar-regional-extents-and-coverage-dataset
https://catalogue.data.govt.nz/dataset/biodiversity-vegetation-bioveg-2007-gis-layer
https://catalogue.data.govt.nz/dataset/biodiversity-vegetation-bioveg-2002-gis-layer
```
https://catalogue.data.govt.nz/organization/waikato-regional-council is the organization which I exported to csv then from csv back to dcat

**Comment by anotheredward** [comment]
@camfindlay We'll investigate and fix this issue before delivering the script :)

**Comment by ebuckley** [comment]
Re tested with a new csv generated after @boredsteve's change.

LGTM shipit :shipit: 

Thanks Steve <3

File: jsonToCsv/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  Tool for downloading an agency's existing data from data.govt in a format they can use to re-submit it via the schema tool.
+
+
+  Options:
+
+    -V, --version          output the version number
+    -a, --agency [agency]  CKAN agency identifier, eg: ministry-of-health
+    -f, --file [fileName]  CSV filename to write to, eg: output.csv, defaults to: <agency-identifer>.csv
+    -u, --url [url]        ckan url, eg: https://catalogue.data.govt.nz, defaults to: https://catalogue.data.govt.nz
+    -h, --help             output usage information
```
Comment (camfindlay) [inline jsonToCsv/README.md]: Add docs for the -r param

**Comment by camfindlay** [comment]
Also tested, working as expected. 

Can docs been updated then will merge and mark as accepted :+1:

**Pull request merged**